addons/hr_holidays/tests/test_accrual_allocations.py

```diff
@@ -1351,7 +1352,7 @@ def test_accrual_carryover_at_allocation(self):
 
         with freeze_time("2024-04-26"):
             allocation._update_accrual()
-        self.assertEqual(allocation.number_of_days, 0, "Allocations not lost on 1st of January, but on allocation date.")
+        self.assertAlmostEqual(allocation.number_of_days, 0.03, 2, "Allocations not lost on 1st of January, but on allocation date.")
```

Comment:
Note that this should change in the 18.0 fw-port as the carryover date shouldn't grant days anymore

Reply:
Noted I'll keep a eye out for this

Reply:
@Bertrand2  Just to be clear, these are the days accumulated AFTER the carryover date. As in if you have a monthly accrual and your carryover is in he middle of the month you only get the half you would be due for the latter half (after the leaves are reset). I don't really understand why we should not accrue them, could you please elaborate on the intent here ?